build/poms/pom-default.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
is this file needed? could the whole `build` folder be gitignored?

Reply:
Probably.